src/blockly-blocks/block-graph-data.js

```diff
@@ -101,7 +101,7 @@ Blockly.Blocks['graph_exceedance'] = {
 }
 Blockly.JavaScript['graph_exceedance'] = function (block) {
   var location = block.getFieldValue('locations')
-  var threshold = block.getFieldValue('threshold') || 0
+  var threshold = Blockly.JavaScript.valueToCode(block, 'threshold', Blockly.JavaScript.ORDER_ATOMIC) || 0;
```

Comment:
Ah, this is because I was torn over whether the threshold should be a block or a field, and so I ended up committing something that was half-and-half...

My personal preference is that all numbers that don't make sense to loop over really ought to be fields. That way there's no question of wrong types, default values are clearly visible, etc.

However, Noah's mockups have generally used blocks.

I think this is fine for now, and maybe we can remember to bring this up later.

Reply:
I agree - I would rather see this as a field.  It's much easier/faster to build a program this way, However, the specs do have a block so we should leave it this way for now.  Regardless, we should bring this up at some point.